src/api/config/feature.yml

```diff
@@ -1,30 +1,26 @@
-production:
-  features: &default
-    image_templates: true
-    kiwi_image_editor: false
-    cloud_upload: false
-    cloud_upload_azure: false
-    bootstrap: false
+# (keep aligned with lib/feature_switch/obs_repository.rb)
+#production:
+#  features:
+#   image_templates: true
+#   cloud_upload: false
+#   cloud_upload_azure: false
+#   bootstrap: false
+#   kiwi_image_editor: false
```

Comment:
Configuration happens in `.yml` files, period. This just won't fly, sorry.

Reply:
that's all good - it's just the defaults that is ruby. If admins want to configure, yaml is the way to go

Reply:
This is a file for admins. You tell them to keep it in sync with the ruby file.

Reply:
where do you tell them @hennevogel ?

Reply:
You don't tell them to keep configuration in sync with some source code file.

Reply:
that sync comment is for developers - I move it to the source file instead to keep in sync with the config file. That also makes more sense.